source/slang-neural-module/neural.slang

```diff
@@ -0,0 +1,13 @@
+module neural;
```

Comment:
put a big notice here that this is under active design and experiment and we may remove or change any or all parts of it at any time. not supposed to be used in product code.